include/libgit4cpp/GitRepository.h

```diff
@@ -134,15 +134,33 @@ class GitRepository
 
     /**
      * Remove all entries from the index under a given directory.
+     *
+     * No files are removed from the filesystem.
+     *
      * \param directory Directory to remove
      * \attention Directory is a relative path within repo_path_
+     * \See remove_files() for hints about file removal
      */
     void remove_directory(const std::filesystem::path& directory);
 
     /**
      * Delete specific files from git index.
+     *
+     * No files are removed from the filesystem.
+     *
+     * They also do not need to exist on the filesystem to be removable from the index.
+     *
+     * To delete files from the repository you can eiter:
+     *
+     * - remove the files from the repository with \c remove_files()
```

Comment:
I do not think the `\c` markup is necessary. Doxygen usually is smart enough to hyperlink other member or nonmember functions that it has in its documentation space.

Reply:
The produced html differs slightly

```diff
-files from the repository with <code><a class="el" href="..." title="...">remove_files()</a></code>
+files from the repository with <a class="el" href="..." title="...">remove_files()</a>
```

but probably 'el' already has monospaced in it. On my browser the difference is ... too small anyhow, I would like to have a more different 'code' font.

I remove the `\c`s.